AndelaOSP/andela-societies-frontend
# [#156998268] Add Verify Activities Buttons

### What does this PR do?
Add verify Buttons on the cards in the verify activities page

#### Description of Task to be completed?
As a user logged in as a society secretary, I should be be able to see buttons to verify or reject activities displayed in my Verify Activities section

GIVEN a society secretary is logged in
WHEN they access the Verified Activities view from the sidebar
THEN they should see cards with a cancel or approve green tick button

#### How to test
Once logged in, navigate to the `verify activities` page, you should be able to see the `Approve` and `Reject` buttons on each activity card listed there. Clicking them verifies that activity

#### What are the relevant pivotal tracker stories?
[156998268](https://www.pivotaltracker.com/n/projects/2154966/stories/156998268)

Screenshots
[Link to screenshots on Drive](https://drive.google.com/open?id=1ZwB-A9UjuqY935t6Bzo0shA7OXtXDlyv)

**Note:**
The *verify activities buttons* will be accessible to only society secretaries and success department members once roles have been implemented

@kimani-ndegwa
